src/language-html/embed/print-vue-script-generic-attribute-value.js

```diff
@@ -0,0 +1,27 @@
+import { getUnescapedAttributeValue } from "../utils/index.js";
+import { formatAttributeValue, shouldHugJsExpression } from "./utils.js";
+
+/**
+ * @typedef {import("../../document/builders.js").Doc} Doc
+ */
+
+/**
+ * @returns {Promise<Doc>}
+ */
+function printVueScriptGenericAttributeValue(textToDoc, print, path) {
+  const { node } = path;
+
+  const value = getUnescapedAttributeValue(node);
+
+  return formatAttributeValue(
+    `type T<${value}> = any`,
+    textToDoc,
+    {
+      parser: "babel-ts",
+      __isVueScriptGeneric: true,
```

Comment:
Do you think we should use a better name for this flag? Plugins like `prettier-plugin-pug` may use it in future, we'll not able to change it.

Reply:
Sounds good.

Reply:
Okay, if we can rename it after, this is fine! Let's leave it as it is. If I need to fix anything else, let me know :)